certbot-ci/certbot_integration_tests/conftest.py

```diff
@@ -32,7 +37,7 @@ def pytest_configure(config):
     """
     if not hasattr(config, 'slaveinput'):  # If true, this is the primary node
         with _print_on_err():
-            config.acme_xdist = _setup_primary_node(config)
+            _setup_primary_node(config)
```

Comment:
I am not a big fan of functions with side effect in Python, since you do not have a way to make clear from the code which function parameter is modified by the invocation of the function (on the contrary of C and similar languages where you can use "pass by value" vs "pass by reference"). I think it is more Pythonic to return a tuple of the generated objects, then apply them explicitly afterwards.